README-CONAN.md

```diff
@@ -1,108 +1,193 @@
+_**Visual Studio Users**_ should use define the Generator and Build Type:
+
+More information about CMake/Exiv2: [README-CMAKE](README-CMAKE).
+
+<name id="1-6"></a>
+##### 1.6) Build Exiv2:
 
-The first thing we need to do, is to check that conan was properly installed and you can access it:
 ```bash
-$ conan --version
-Conan version 1.1.1
+$ cmake --build . --config Release
 ```
 
-The first time you run conan, it will auto-detect your configuration and store it in a default profile in the file
-$HOME/.conan/profiles/default. In case you are using Windows, and have a installation of Visual Studio 2017, the default
-profile will look like this:
+[TOC](#TOC)
+
+<name id="2"></a>
+## 2) Platform Notes
 
-```Ini
+<name id="2-1"></a>
+### 2.1) Linux Notes
+
+##### Default Profile
+
+When you run conan install for the first time, it will detect and write the default profile ~/.conan/profile/default.  On my Ubuntu system with GCC 4.9, this is:
+
+```bash
+[settings]
+os=Linux
+os_build=Linux
+arch=x86_64
+arch_build=x86_64
+compiler=gcc
+compiler.version=4.9
+compiler.libcxx=libstdc++
+build_type=Release
+[options]
+[build_requires]
+[env]
+```
+
+##### Changing profile settings
+
+One of the most important **profile** settings to be adjusted in your conan profile when working on Linux is the field
+
+```bash
+compiler.libcxx=libstdc++11  # Possible values: libstdc++, libstdc++11, libc++
+```
+With the arrival of the c++11 standard, and the growing popularity of the *clang* compiler, it is increasingly important which version of the standard library to use (corresponds to the `-stdlib` compiler flag).
+
+Recommended **libcxx**
+settings works with conan and different compilers:
+
+```bash
+compiler.libcxx=libstdc++11  # will use -stdlib=libstdc++ and define _GLIBCXX_USE_CXX11_ABI=1
+compiler.libcxx=libstdc++    # will use -stdlib=libstdc++ and define _GLIBCXX_USE_CXX11_ABI=0
+compiler.libcxx=libc++       # will use -stdlib=libc++
+
+```
+
+As a rule of thumb, set `compiler.libcxx=libstdc++11` when using a version of gcc >= 5.1.
+
+More information about the standard library and GCC [dual ABI in gcc](https://gcc.gnu.org/onlinedocs/libstdc++/manual/using_dual_abi.html) with GCC works.
+
+Please, be aware that normally when using gcc >= 5.1, \_GLIBCXX\_USE\_CXX11\_ABI is set to 1 by default. However some linux
+distributions might set that definition to 0 by default. In case you get linking errors about standard c++ containers or
+algorithms when bringing the Exiv2 dependencies with conan, this might indicate a mismatch between the value set in
+**compiler.libcxx** and the default values used in your distribution.
+
+<name id="2-2"></a>
+### 2.2) Visual Studio Notes
+
+### Profiles for Visual Studio
+
+You can build Exiv2 with either Visual Studio 2015 (version 14) or 2017 (version 15).  You create profiles in %HOMEPATH%\.conan\profiles with a text editor.
+
+If you have an installation of Visual Studio 2017, the profile msvc2017Release64 is as follows:
```

Comment:
I saw few mentions to the profile name `msvc2017Release64`. As it is written, it looks like we have to use that name, but users can use the profile name they prefer. Maybe it would be good to clarify that.